src/test/java/service/file/FileWriterTest.java

```diff
@@ -0,0 +1,43 @@
+package service.file;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterTest {
+    public static final String GOOD_PATH = "src\\test\\resources\\report.csv";
+    public static final String BAD_PATH = "bad\\path.csv";
+    public static final String DATA = "bad data too";
+    public static final String GOOD_DATA = "b, banana, 100\nh, malina, 300\n";
+    public static final List<String> EXPECTED_LIST = List.of("b, banana, 100", "h, malina, 300");
+    private static FileWriter fileWriter = new FileWriterImpl();
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void writeToFile_BadPath_Not_Ok() {
+        fileWriter.writeToFile(BAD_PATH, DATA);
+    }
+
+    @Test
+    public void writeToFile_GoodPath_Ok() {
+        File file = new File(GOOD_PATH);
+        fileWriter.writeToFile(file.getPath(), GOOD_DATA);
+        List<String> expected = EXPECTED_LIST;
+        try {
+            List<String> actual = Files.readAllLines(file.toPath());
+            assertEquals(expected, actual);
+            Files.write(file.toPath(), "".getBytes());
```

Comment:
why do we need to write afterwards?

Reply:
Cleaning file, don't need data in it afterwards anyway. Will delete this for now, don't know where else can i put it or if i need it.

Reply:
Ok. if you you have only one file - it's fine. But you can do this in `AfterClass` in this situation, or `After`, depends on circumstances.

Reply:
okay, will remember, thanks. : )